.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: rust
+rust:
```

Comment:
Hmm, looking at the travis log it appears only nightly is tested now. I think having `1.1.0` is necessary.